web/templates/pages/members.html

```diff
@@ -20,13 +19,13 @@
 			<ul class="about_list">
 			{% for member in members_list %}
 				{% if member.activity_status == 'MC' %}
-					<li class="mentor-cat"><a href="{{member.githubuser}}"><img src="../static/img/{{member.profile_picture}}">{{member.fullname}}</a> (Mentor cat)<li>
+					<li class="mentor-cat"><a href="{{ member.githubuser }}"><img src="{{ MEDIA_URL }}{{ member.profile_picture.url }}" alt="{{ member.fullname }}" />{{ member.fullname }}</a> (Mentor cat)<li>
```

Comment:
Have you actually tried this link, does it work? (same goes for all others link that should be going to the member's GitHub page)